apps/infrastructure/src/cloudfront.tf

```diff
@@ -90,12 +90,79 @@ resource "aws_cloudfront_distribution" "website" {
   }
 
   viewer_certificate {
-    cloudfront_default_certificate = var.use_custom_certificate ? false : true
+    cloudfront_default_certificate = true
     # acm_certificate_arn            = var.use_custom_certificate ? aws_acm_certificate.website.arn : null
     # ssl_support_method             = var.use_custom_certificate ? "sni-only" : null
   }
 }
 
+resource "aws_cloudfront_distribution" "cdn_multi_website" {
+  for_each = var.websites_configs
+
+  origin {
+    domain_name = aws_s3_bucket.website.bucket_regional_domain_name
+    origin_id   = aws_s3_bucket.website.bucket
+    s3_origin_config {
+      origin_access_identity = aws_cloudfront_origin_access_identity.main.cloudfront_access_identity_path
+    }
+    origin_path = each.value.origin_path
+  }
+
+  enabled             = true # enable CloudFront distribution
+  is_ipv6_enabled     = true
+  comment             = "CloudFront distribution for the static website."
+  default_root_object = "index.html"
+
+  # to be uncommented when available dns domain and certificate is validated
```

Comment:
**question**: what should be uncommented?

Reply:
comment removed in next commit